experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -465,6 +467,11 @@ export class FetchInstrumentation extends InstrumentationBase<
    * implements enable function
    */
   override enable(): void {
+    if (isNode) {
+      // Node.js v18+ *does* have a global `fetch()`, but this package does not
+      // support instrumenting it.
+      return;
```

Comment:
I'm wondering - should we warning (or info) log here? :thinking:

Reply:
Yah, I wasn't sure. I'll add one. I suppose it is user-error -- i.e. something worth warning about -- if they supply instrumentation-fetch for usage in Node.js.

Reply:
Added in commit 6211772d. It looks like:

```
% OTEL_LOG_LEVEL=warn node foo.js
@opentelemetry/instrumentation-fetch this instrumentation is intended for web usage only, it does not instrument Node.js's fetch()
...
```

Reply:
looks good, thanks! :slightly_smiling_face: